src/PerfView/Dialogs/RunCommandDialog.xaml

```diff
@@ -301,8 +301,12 @@
                     <Hyperlink Command="Help" CommandParameter="JITInliningCheckBox">JIT Inlining:</Hyperlink>
                     </TextBlock>
                     <CheckBox Grid.Row="0" Grid.Column="9" Name="JITInliningCheckBox" VerticalAlignment="Center" />
-                    
-                    <CheckBox Grid.Row="0" Grid.Column="11" Name="Unused1" Visibility="Hidden"  VerticalAlignment="Center"/>
+
+                    <TextBlock Grid.Row="0" Grid.Column="10" VerticalAlignment="Center" HorizontalAlignment="Right" Margin="8,2,5,2" 
+                                ToolTip="Log information about CCW reference counting.">
```

Comment:
Instead of just CCW say 

NET Native Com Callable Wrapper (CCW)

Reply:
done